content/cif-connector/tests/webpack.config.js

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************************
```

Comment:
Webpack is required only if you wrote Javascript code which uses features that are not supported by the browsers (e.g. module related stuff - `export`, `module.exports`, `import` etc). Looking at the two client library fies I see no such code, so we don't need webpack here.